data/gui/window/lobby_main.cfg

```diff
@@ -375,6 +375,18 @@
 				[/toggle_button]
 			[/column]
 
+			[column]
+				grow_factor = 0
+				border = "all"
+				border_size = 5
+
+				[toggle_button]
+					definition = "default"
+					id = "filter_auto_hosted"
+					label = _ "Auto-hosted games"
```

Comment:
Needs a `# po:` comment telling the translators what this means.

I don't have a suggestion for a clearer English wording, but it doesn't seem clear in English either.

Reply:
I suggest "Bot-hosted".

Reply:
My concern would just be any sort of negative connotation associated with calling them bot-hosted. Maybe I'm concerned over nothing though.